fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -9877,7 +9927,7 @@
 | Test Case | Flags | Result |
 | --- | --- | --- |
 | `src=(string)`| (changed)| `"https://reactjs.com/"` |
-| `src=(empty string)`| (changed)| `"http://localhost:3000/"` |
+| `src=(empty string)`| (initial, warning)| `<empty string>` |
```

Comment:
Logs `An empty string ("") was passed to the src attribute. This may cause the browser to download the whole page again over the network. To fix this, either do not render the element at all or pass null to src instead of an empty string.`.

I think this was added intentionally?

Reply:
It was. Will land `enableFilterEmptyStringAttributesDOM` on main next.